lib/index.js

```diff
@@ -74,6 +74,12 @@ function getFrameMethodName (frame, funcName) {
     return typeName + '.' + (methodName || '<anonymous>');
 }
 
+function processFrames (stackFrames, processFn) {
+    return stackFrames.map(function (frame) {
+        return processFn(frame);
+    });
```

Comment:
It can be `stackFrames.map(processFn)`. Move `processFn` check in this function.